package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "react-datetime",
-  "version": "0.4.0",
+  "version": "0.4.3",
```

Comment:
I think this is only a diff because I rebased on top of the upstream master, so this is a change from the previous commit, even though it's not a change from your master... Weird...